### PR TITLE
Feature/artifacts

### PR DESCRIPTION
**Pull request by rcordeau**
Removing submodule and adding Artifactory support and Common/Schema artifact support

---
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_reporting/125)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -137,16 +175,34 @@ subprojects {
...
+    into schemaBaseDir
 }
 
-task cleanReportingTest(type: GradleBuild) {
-    group = "Schema"
-    description = "Migrates the schemas that integration tests depend on"
-    dir = "RDW_Schema"
-    tasks = ["cleanReporting_test"]
+//These are the tasks available in the Schema dependency that we would want to use in this project
+def taskMap = ["migrateReporting": "Migrates the schemas that are required for running locally",
```
Comment (malaffoon) [inline build.gradle]: Can/should we remove `Reporting` from all these? This is in the reporting project, it better be doing the work for reporting.

Comment (rcordeau) [inline build.gradle]: Yeah we could do that.

Comment (rcordeau) [inline build.gradle]: Well, maybe not. The specific schema related clean/migrate tasks are part of the schema project, like cleanWarehouse, cleanReporting, cleanStaging, and cleanAll. Are you looking to remove the need to type migrateReporting on the command line?

Comment (malaffoon) [inline build.gradle]: It's not critical, i was just imagining myself typing `RDW_Reporting$ gradle migrateReporting` and thinking to myself, "what the hell else would i be migrating here?"

Comment (rcordeau) [inline build.gradle]: I could change it to migrateSchema, or migrateDb, or...? Any desires?

Comment (malaffoon) [inline build.gradle]: Nah, it's fine the way it is.

Comment (rcordeau) [inline build.gradle]: ok

File: build.gradle [hunk 2]
```diff
@@ -17,13 +17,50 @@ buildscript {
...
 ext['thymeleaf.version'] = '3.0.6.RELEASE'
 ext['thymeleaf-layout-dialect.version'] = '2.0.0'
 
 
 apply plugin: 'io.spring.dependency-management'
 
+// JFrog artifactory setup
+
+artifactory {
+    contextUrl = "https://airdev.jfrog.io/airdev"
```
Comment (bgsavage) [inline build.gradle]: Check it out, we are legit.  :) 
https://airdev.jfrog.io/airdev/webapp/#/artifacts/browse/simple/General/libs-releases/org/opentestsystem/rdw

**Pull request merged**